src/plugins/vis_augmenter/public/vega/helpers.test.ts

```diff
@@ -409,19 +408,6 @@ describe('helpers', function () {
         )
       ).toStrictEqual(TEST_DATATABLE_SINGLE_VIS_LAYER_EMPTY);
     });
-    // below case should not happen since only VisLayers with a populated
-    // set of events should be passed from the plugins. but, if it does
-    // happen, we can handle it more gracefully instead of throwing an error
-    it('vis layer with empty events adds nothing to datatable', function () {
```

Comment:
Why are we removing this test? Is it because it would not be empty and have placeholder `0` values? If so, we should modify the test for that.

Reply:
The test is invalid now, since it was testing that nothing would be added to the datatable. But now we add 0s. The rest of the tests handle such scenarios

Reply:
Additionally the test was hacky to begin with. The scenario should never happen and other tests to ensure the logic should never reach this state are already tested.